Library/Homebrew/cask/lib/hbc/cli/style.rb

```diff
@@ -11,7 +11,7 @@ def self.help
 
       def run
         install_rubocop
-        system "rubocop", *rubocop_args, "--", *cask_paths
+        system({ "XDG_CACHE_HOME" => HOMEBREW_CACHE }, "rubocop", *rubocop_args, "--", *cask_paths)
```

Comment:
Strip all environment variables from any `brew` subprocess that haven't been whitelisted into `HOMEBREW_` variables. You can try it out and see.